runelite-client/src/main/java/net/runelite/client/plugins/slayer/SlayerPlugin.java

```diff
@@ -338,22 +338,26 @@ public void onVarbitChanged(VarbitChanged varbitChanged)
 		{
 			clientThread.invokeLater(this::updateTask);
 		}
-		else if (varbitId == Varbits.SLAYER_POINTS)
+		else if (varbitId == Varbits.SLAYER_POINTS || varbitId == Varbits.SLAYER_TASK_STREAK || varbitId == Varbits.SLAYER_WILDERNESS_TASK_STREAK || varbitId == Varbits.SLAYER_MASTER)
```

Comment:
This can be formatted like the if-clause above for improved readability.

Reply:
That's very true; accidentally missed this.